lib/resources/example_json/r4_examples_allergy_intolerance.rb

```diff
@@ -113,5 +113,73 @@ module Resources
         "resource": R4_ALLERGY_INTOLERANCE_ENTRY
       ]
     }
+
+    R4_ALLERGY_INTOLERANCE_CREATE ||= {
+      "resourceType": "AllergyIntolerance",
+      "contained":[
+        {
+          "resourceType": "PractitionerRole",
+          "id": "638995",
+          "code":[
+            {
+              "coding":[
+                {
+                  "system": "http://terminology.hl7.org/CodeSystem/practitioner-role",
+                  "code": "nurse"
+                }
+              ],
+              "text": "Nurse"
+            }
+          ]
+        }
+      ],
+      "clinicalStatus":{
+        "coding":[
+           {
+              "system": "http://terminology.hl7.org/CodeSystem/allergyintolerance-clinical",
+              "code": "active"
+           }
+        ]
+      },
+      "verificationStatus":{
+        "coding":[
+           {
+              "system": "http://terminology.hl7.org/CodeSystem/allergyintolerance-verification",
+              "code": "confirmed"
+           }
+        ]
+      },
+      "type": "allergy",
+      "category":[
+        "environment",
+        "food",
+        "medication"
+      ],
+      "criticality": "high",
+      "code":{
+        "text": "Demeclocycline - substance"
+      },
+      "patient":{
+        "reference": "Patient/1316020"
+      },
+      "encounter":{
+        "reference": "Encounter/4749909"
+      },
+      "onsetDateTime": "2019-09-11T05:00:00.000Z",
+      "asserter":{
+        "reference": "#638995"
+      },
+      "reaction":[
+        {
+          "manifestation":[
+            {
```

Comment:
Need to wrap this inside "coding"

Reply:
https://github.com/cerner/fhir.cerner.com/pull/324/commits/8add7576d67f10e24cfb8b8da2c99496a34ed407#diff-c2b897bdc8e9690677a21039c233a57fR176